src/k8s-engine/lima.ts

```diff
@@ -453,6 +455,27 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     }
   }
 
+  protected updateConfigPortForwards(config: LimaConfiguration) {
+    let allPortForwards: Array<Record<string, any>> | undefined = config.portForwards;
+
+    if (!allPortForwards) {
+      // This shouldn't happen, but fix it anyway
+      config.portForwards = allPortForwards = DEFAULT_CONFIG.portForwards ?? [];
+    }
+    const dockerPortForwards = allPortForwards?.find(pair => Object.keys(pair).length === 2 &&
```

Comment:
If someone explicity changes the config to put `hostSocket` at a different location, I would respect that and leave it. If the difference is due to corruption of the config file, we're wading into an area where software providers often will recommend shutting down, deleting the config file, and restarting.

Reply:
Users should not need to modify the config file. They should move the whole `lima` directory to a different location, if they have to. This could even be to a separate disk, in case there is not enough free space on the host boot disk for the VM (grows to 100GB).